_assets/ci/Jenkinsfile

```
@@ -47,19 +52,13 @@ pipeline {
           stages {
             stage('Compile') { steps { dir(env.STATUS_PATH) {
               sh 'make statusgo-android'
+              sh "cp build/bin/statusgo-android-16.aar ${dest}/status-go-android-${lib.suffix()}.aar"
             } } }
             stage('Archive') { steps {
-              sh """
-                cp ${env.STATUS_PATH}/build/bin/statusgo-android-16.aar \
-                  ${env.WORKSPACE}/status-go-android-${lib.suffix()}.aar
-              """
-              archiveArtifacts("status-go-android-${lib.suffix()}.aar")
+              archiveArtifacts("pkg/status-go-android-${lib.suffix()}.aar")
```

```suggestion
              archiveArtifacts("${dest}/status-go-android-${lib.suffix()}.aar")
```

I think this can be changed in quite a few places here.

---

Thought that too, but apparently absolute paths screw with `archiveArtifacts` and break it.

---

Maybe we should add a warning comment so that the next guy isn't tempted to "fix it" then.